lib/foreman_ansible_core/ansible_runner_task_launcher.rb

```diff
@@ -0,0 +1,165 @@
+require 'fileutils'
+
+module ForemanAnsibleCore
+  class AnsibleRunnerTaskLauncher < ForemanTasksCore::TaskLauncher::AbstractGroup
+    class AnsibleRunnerRunner < ForemanTasksCore::Runner::Parent
```

Comment:
I would help readability to have the `AnsibleRunnerRunner` in separate file (the methods between `AnsibleRunnerTaskLauncher` and `AnsibleRunnerRunner` might mix quite easily, as it's not very visible where one class begins and other ends.

Reply:
Also, perhaps we could have module `ForemanAnsibleCore::Runners`, where there would be both `AnsibleRunner` and `Playbook`, this way, we would get rid of the `AnsibleRunnerRunner`, that looks like a typo